### PR TITLE
Feature missing

### PR DESCRIPTION
**Pull request by clarkfitzg**
This doesn't work yet, just putting it out there so you can see the test.

### CODE REVIEW

File: R/ast_node.R [hunk 1]
```diff
@@ -590,6 +590,22 @@ Parameter = R6::R6Class("Parameter", inherit = Symbol,
...
+Missing = R6::R6Class("Missing", inherit = Symbol,
+  "public" = list(
+    basename = NULL,
+    ssa_number = NULL,
+    namespace = NULL,
+    namespace_fn = NULL,
+
+    initialize = function(...)
+    {
+      super$initialize("", ...)
```
Comment (clarkfitzg) [inline R/ast_node.R]: Maybe could use special symbol, ie. `super$initialize(".missing", ...)

**Comment by clarkfitzg** [comment]
Actually looks like I was misunderstanding what the `node_apply` does. I should be using `astTraverse`.

**Comment by clarkfitzg** [comment]
Now this is just a change to the docs. If you like the Makefile keep it, otherwise I'll take it out. I can also rebase.

**Comment by nick-ulle** [comment]
I think it makes sense to have a specific class for missing values, since they arise in calls and also in default arguments. So I've modified and merged part of this PR in 731d3bc5c7319ba332935f204c2d24da23447663.

As for `node_apply`, I plan to remove it and recommend `replace_nodes` instead.

**Pull request closed without merging**